src/layout/Stepper/Stepper.tsx

```diff
@@ -36,57 +54,65 @@ export const Stepper: FC<StepperProps> = ({
   const totalSteps = childrenStepProps.length - 1;
 
   return (
-    <div className={theme.base}>
+    <MotionGroup
+      className={cn(theme.base, className)}
+      initial={animated ? 'initial' : null}
```

Comment:
How does this animate? You don't pass the directions.

Reply:
This is for animation disabling optionally